includes/class-wc-shortcodes.php

```diff
@@ -49,6 +49,71 @@ public static function init() {
 
 		// Alias for pre 2.1 compatibility.
 		add_shortcode( 'woocommerce_messages', __CLASS__ . '::shop_messages' );
+
+		if ( function_exists( 'register_block_type' ) ) {
+			foreach ( $shortcodes as $shortcode => $function ) {
```

Comment:
That's why we don't need just one product shortcode :P

Reply:
Hrm?  I removed the grand iterator figuring it'd be better to just declare the ones we care about directly.